be/src/olap/delta_writer.cpp

```diff
@@ -231,7 +231,26 @@ int64_t BaseDeltaWriter::mem_consumption(MemType mem) {
     return _memtable_writer->mem_consumption(mem);
 }
 
-void DeltaWriter::_request_slave_tablet_pull_rowset(const PNodeInfo& node_info) {
+Status safe_get_file_size(const std::string& file_path, int64_t* file_size) {
+    if (file_size == nullptr) {
+        return Status::InvalidArgument("Null output parameter in safe_get_file_size");
```

Comment:
I think you can just use `CHECK(file_size != nullptr) << "Null output parameter in safe_get_file_size"`